e2e/pages/uploadDocuments/uploadDocuments.js

```diff
@@ -50,4 +44,9 @@ module.exports = {
     I.fillField(this.fields.otherDocumentsTitle_2, 'Document');
     I.attachFile(this.documents.otherDocuments_2, file);
   },
+
+  socialWorkChronologyToFollow() {
```

Comment:
I think the function needs to be renamed something like 'selectingSocialWorkChronologyToFollow' that's very long though 🤔

Reply:
More `selectSocialWorkChronologyToFollow` ;)